src/meds_torch/models/zero_shot_labeler/time_to_event_labeler.py

```diff
@@ -0,0 +1,140 @@
+from dataclasses import dataclass
+
+import torch
+
+from meds_torch.models.zero_shot_labeler.utils import TrajectoryBatch
+
+
+@dataclass
+class TaskLabeler:
```

Comment:
Is this a general class or for a specific kind of task only? If the latter, I suggest a more specific name.